eth/protocols/eth/protocol.go

```diff
@@ -33,9 +33,9 @@ const (
 	ETH66 = 66
 )
 
-// ProtocolName is the official short name of the `eth` protocol used during
+// ProtocolName is the official short name of the `quai` protocol used during
 // devp2p capability negotiation.
-const ProtocolName = "eth"
+const ProtocolName = "quai"
```

Comment:
Should this conform to the new const naming standard?
(eg `const_ProtocolName`)

Reply:
We can discuss if this is absolutely necessary.

Reply:
Made the change